ffi/diplomat/src/segmenter_line.rs

```diff
@@ -50,15 +50,18 @@ pub mod ffi {
     }
 
     #[diplomat::opaque]
-    #[diplomat::rust_link(icu::segmenter::LineBreakIteratorPotentiallyIllFormedUtf8, Typedef)]
+    #[diplomat::rust_link(icu::segmenter::LineBreakIterator, Struct)]
+    #[diplomat::rust_link(icu::segmenter::LineBreakIteratorPotentiallyIllFormedUtf8, Typedef, compact)]
```

Comment:
thought: ah, i see the complexity here.

I think given that the typedefs are all underdocumented it makes sense to link to LineBreakIterator directly, but I'm wondering if we'd benefit from having better documentation on the typedefs (even just "A [LineBreakIterator] that accepts potentially ill formed utf-8. See [LineBreakIterator] for more info.") would be better.

Then we can have a single non-hidden rust_link here.

Reply:
That's a good idea, although I still need to link the `LineBreakIterator` from something.